employees/urls.py

```diff
@@ -37,4 +37,9 @@
         views.ExportReportsInProjectView.as_view(),
         name="export-project-data-xlsx",
     ),
+    url(
+        r"^export/project/(?P<pk>[0-9]+)/author/(?P<user_pk>[0-9]+)/reports(?P<year>[0-9]{4})/(?P<month>[0-9]{1,2})/$",
+        views.ExportAuthorReportProjectView.as_view(),
+        name="export-project-author-reports-data-xlsx",
```

Comment:
I think those names are no longer adequate because we also allow to export `csv` with the same url. Would it be possible to fix all of them in this PR? If no, please fix just this one and add new issue for fixing rest of `-xlsx` urls.

Reply:
I will change it now :)